sdk/examples/how_tos/nfts/mint_nft.rs

```diff
@@ -57,13 +55,15 @@ async fn main() -> Result<()> {
         .set_stronghold_password(std::env::var("STRONGHOLD_PASSWORD").unwrap())
         .await?;
 
+    let metadata = Irc30Metadata::new("My NFT", "MYNFT", 10).with_description("My super cool NFT.");
```

Comment:
Irc30Metadata is for foundry outputs metadata, not NFTs

Reply:
Oh 😆